tests/route/test_default_route.py

```diff
@@ -176,7 +178,7 @@ def test_default_route_with_bgp_flap(duthosts, tbinfo):
     # Get uplink namespaces/asics for multi-asic
     if duthost.is_multi_asic:
         bgp_name_to_ns_mapping = duthost.get_bgp_name_to_ns_mapping()
-        uplink_ns = get_uplink_ns(tbinfo, bgp_name_to_ns_mapping)
+        uplink_ns = get_uplink_ns(tbinfo, bgp_name_to_ns_mapping, config_facts['DEVICE_NEIGHBOR_METADATA'])
```

Comment:
`def get_uplink_ns` only has 2 arguments, should we add a default argument in the function definition?